### DISCUSSION TITLE
Including an HTML file

### DISCUSSION THREAD

**Discussion by FPA-AkshayGollahalli** [vuejs/vitepress#4236] [Q&A]
I have a generated html file from Jupyter notebook, is it possible to include it as how we can include a markdown file?

**Comment by mil7** [comment]
Hi there,

I had a similar issue. My solution was to [Linking to Non-VitePress pages](https://vitepress.dev/guide/routing#linking-to-non-vitepress-pages). The linked page is not really _included_ as such but for me linking to an internal source without triggering the [conversion to a router link](https://vitepress.dev/guide/markdown#links) was gut enough.
